client/src/themes/theme.ts

```diff
@@ -1,10 +1,23 @@
 import { createTheme } from '@mui/material/styles';
+import '@mui/material/styles';
+
+declare module '@mui/material/styles' {
+  interface Palette {
+    Green?: Palette['primary'];
+  }
+  interface PaletteOptions {
+    Green?: PaletteOptions['primary'];
+  }
+}
 
 export const theme = createTheme({
   palette: {
     primary: {
       main: '#f14140',
     },
+    Green: {
+      main: '#008000',
+    },
```

Comment:
First of all, good job on augmenting the mui theme properly 🙌🏼 

The issue here is that `Green` doesn't really contribute to our theme palette. Every key in our palette has an approximate role to play as outlined [here](https://mui.com/customization/palette/#palette-colors), though these aren't very strict roles. It still makes sense to augment the theme palette only when there is a new role to be filled in our design - an example would be to add a `tertiary` color if our design used a tri-color palette.

The other side of this coin would be to think about the role of the UI elements rather than to assign a direct color to it. Is it a primary element that needs to be visible? If so the `primary` color or one of its variants would be suitable. If it's not the most important element, we can maybe use `secondary` or `info` as its color. If it's an `error` or a `warning`, we can apply the respective theme colors for those, so on. In our case, the notification badge would be something we want to highlight, so it makes the most sense to assign the `primary` color to it.

Lastly, my initial sugesstion: 
> I will mention though I don't advise using theme/pallete properties outside of their intended use cases (e.g. using text.disabled for a darker background) as these can be hard to identify gotchas when changes come through in the future. I would recommend adding a color to your palette instead (e.g. theme.pallete.background.darkerPaper: grey[100]). 👍🏼

I think implementing this color in `useStyles` file as you have done is a fine alternative. And with that, we don't really need to augment the theme at this stage 👍🏼

Reply:
The mockups show the notification badge as being green, and our primary color is orange, so does that mean I should just use primary here then? I don't think a green color is going to be our secondary or tertiary color, so I can't use these either. And from the earlier comment you mentioned to not use success when I originally applied it to the badge because it's not really a success message or popup which makes sense. 

From what I see in the MUI Pallette docs here (https://mui.com/customization/palette/) , the other properties of your primary or secondary color are just different shades of those colors, so I can't add a green color to these.

So I just did use styles with a badge property and changed it there, which I think is okay?

Reply:
@rj13371 Yes, we can implement a color in the `useStyles` file. As for the color of the badge we can go with either 'green' or 'orange', imo I think green looks better 😄

Reply:
Ah @rj13371, I'm sorry I hadn't noticed the mocks had green badges 🙈 

Apologies for the confusion 🙃

Reply:
Okay the badge is now green :)